### PR TITLE
docs: update reference design page

### PR DESCRIPTION
**Pull request by NilaySener**
## Description

Updated information of sensors and systems has been added to the Reference HW section.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: docs/how-to-guides/integrating-autoware/creating-maps/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This page explains how users can create maps that can be used for Autoware.
 
 Traditionally, a Mobile Mapping System (MMS) is used in order to create highly accurate large-scale point cloud maps. However, since a MMS requires high-end sensors for precise positioning, its operational cost can be very expensive and may not be suitable for a relatively small driving environment. Alternatively, a Simultaneous Localization And Mapping (SLAM) algorithm can be used to create a point cloud map from recorded LiDAR scans. Some of the useful open-source SLAM implementations are listed in this [page](open-source-slam/index.md).
 
-If you prefer proprietary software that is easy to use, you can try a fully automatic mapping tool from [MAP IV, Inc.](https://www.map4.jp/), [_MapIV Engine_](https://www.map4.jp/solutions/mapping_localization/map4engine/). They currently provide a trial license for Autoware users free of charge.
+If you prefer proprietary software that is easy to use, you can try a fully automatic mapping tool from [MAP IV, Inc.](https://www.map4.jp/), [_MapIV Engine_](https://www.map4.jp/solutions/mapping-localization/map4-engine/). They currently provide a trial license for Autoware users free of charge.
```
Comment (armaganarsln) [inline docs/how-to-guides/integrating-autoware/creating-maps/index.md:11]: This is not part of the reference HW page. Why did you change this one?

**Comment by armaganarsln** [comment]
apart from changes in the how to create maps the document looks good.

**Review by armaganarsln — CHANGES_REQUESTED** [review]
apart from changes in the how to create maps the document looks good.

File: docs/how-to-guides/integrating-autoware/creating-maps/index.md (hunk 1 above)
Comment (NilaySener) [reply docs/how-to-guides/integrating-autoware/creating-maps/index.md]: @armaganarsln Hi thank you for the review, i think there was a confusion and it was fixed in the last commit.

File: docs/reference-hw/cameras.md [hunk 2]
```diff
@@ -42,3 +51,36 @@ Link to ROS 2 driver:
 
 Link to company website:  
 [https://www.alliedvision.com/en/products/camera-series/mako-g](https://www.alliedvision.com/en/products/camera-series/mako-g)
+
+## **Tier IV Cameras**
```
Comment (mitsudome-r) [inline docs/reference-hw/cameras.md:55]: TIER IV is all capital case
```suggestion
## **TIER IV Cameras**
```

File: docs/reference-hw/cameras.md [hunk 3]
```diff
@@ -42,3 +51,36 @@ Link to ROS 2 driver:
...
+
+![images/camera-tieriv.png](images/camera-tieriv.png)
+
+TierIV cameras which has ROS 2 driver and tested by one or more community members are listed below:
```
Comment (mitsudome-r) [inline docs/reference-hw/cameras.md:59]:
```suggestion
TIER IV cameras which has ROS 2 driver and tested by one or more community members are listed below:
```

File: docs/reference-hw/ad_sensor_kit_suppliers.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+## **Tier IV AD Kit**
+
+![images/ad_kit-tieriv.png](images/ad_kit-tieriv.png)
+
+Tier IV sensor fusion system contents are listed below:
+
+| Supported Products List | Camera         | Lidar                            | ECU                        | ROS2 Support | Autoware Tested (Y/N) |
+| ----------------------- | -------------- | -------------------------------- | -------------------------- | ------------ | --------------------- |
+| Tier IV ADK             | TIER IV C1, C2 | HESAI (AT-128,XT-32)<br>Velodyne | ADLINK (RQX-58G, AVA-3510) | Y            | Y                     |
```
Comment (mitsudome-r) [inline docs/reference-hw/ad_sensor_kit_suppliers.md:24]:
```suggestion
| TIER IV ADK             | TIER IV C1, C2 | HESAI (AT-128,XT-32)<br>Velodyne | ADLINK (RQX-58G, AVA-3510) | Y            | Y                     |
```

File: docs/reference-hw/ad_sensor_kit_suppliers.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+Link to company website:  
+[https://leodrive.ai/](https://leodrive.ai/)
+
+## **Tier IV AD Kit**
+
+![images/ad_kit-tieriv.png](images/ad_kit-tieriv.png)
+
+Tier IV sensor fusion system contents are listed below:
+
+| Supported Products List | Camera         | Lidar                            | ECU                        | ROS2 Support | Autoware Tested (Y/N) |
```
Comment (mitsudome-r) [inline docs/reference-hw/ad_sensor_kit_suppliers.md:22]:
```suggestion
| Supported Products List | Camera         | Lidar                            | ECU                        | ROS 2 Support | Autoware Tested (Y/N) |
```

**Comment by mitsudome-r** [comment]
Thanks for the PR!
I did made a review on TIER IV related sections. They seem general good in terms of context. I made some small comments about the company name.

**Comment by evshary** [comment]
@NilaySener I've updated the description of ADLINK IPC in this branch.
https://github.com/NilaySener/autoware-documentation/pull/1

**Comment by xmfcx** [comment]
@armaganarsln could you approve if the changes you've requested are applied?

**Review by armaganarsln — APPROVED** [review]
Thank you for your work. We could release it as the first version and update it next year with new information.
